### PR TITLE
merged formatting changes and removed while loops in asynchronous client

### PR DESCRIPTION
**Pull request by egimaben**

### CODE REVIEW

File: spring-core/src/test/java/com/baeldung/factorybean/FactoryBeanXmlConfigTest.java [hunk 1]
```diff
@@ -7,15 +7,22 @@
...
 import org.springframework.beans.factory.annotation.Autowired;
+>>>>>>> f354f9e3057481c76c10a1e2aa127d3abb95b329
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration(locations = { "classpath:factorybean-spring-ctx.xml" })
 public class FactoryBeanXmlConfigTest {
 
+<<<<<<< HEAD
```
Comment (pivovarit) [inline spring-core/src/test/java/com/baeldung/factorybean/FactoryBeanXmlConfigTest.java]: :(

File: spring-core/src/test/java/com/baeldung/factorybean/FactoryBeanJavaConfigTest.java [hunk 2]
```diff
@@ -11,14 +23,21 @@
 import static org.hamcrest.core.IsEqual.equalTo;
 import static org.junit.Assert.assertThat;
 
+>>>>>>> f354f9e3057481c76c10a1e2aa127d3abb95b329
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration(classes = FactoryBeanAppConfig.class)
 public class FactoryBeanJavaConfigTest {
 
+<<<<<<< HEAD
```
Comment (pivovarit) [inline spring-core/src/test/java/com/baeldung/factorybean/FactoryBeanJavaConfigTest.java]: Not really merged :(

File: spring-core/src/main/java/com/baeldung/factorybean/FactoryBeanAppConfig.java [hunk 3]
```diff
@@ -6,11 +6,23 @@
 @Configuration
 public class FactoryBeanAppConfig {
     
+<<<<<<< HEAD
+    @Bean
+=======
     @Bean(name = "tool")
+>>>>>>> f354f9e3057481c76c10a1e2aa127d3abb95b329
```
Comment (pivovarit) [inline spring-core/src/main/java/com/baeldung/factorybean/FactoryBeanAppConfig.java]: :(

Comment (egimaben) [inline spring-core/src/main/java/com/baeldung/factorybean/FactoryBeanAppConfig.java]: @pivovarit  damn, let me work on them... this has been quite common on my pulls. Tell me how I can pull and update my local repo without adding merge conflicts :)

Comment (egimaben) [inline spring-core/src/main/java/com/baeldung/factorybean/FactoryBeanAppConfig.java]: @pivovarit just updated the PR

**Pull request merged**